### DISCUSSION TITLE
访问https://cn.vuejs.org/guide/quick-start经常遇到网页版面错乱

### DISCUSSION THREAD

**Discussion by acengtb** [vuejs-translations/docs-zh-cn#931] [Q&A] [answered]
访问 [cn.vuejs.org ](https://cn.vuejs.org/guide/quick-start) 时，时不时遇到 网页版面混乱，查看 chrome 控制台，提示：
Refused to apply style from 'https://cn.vuejs.org/assets/style.C0EEGlWo.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
这是怎么回事吗？

**Comment by acengtb** [comment]
补充，环境如下：
浏览器  chrome Version 124.0.6367.119 (Official Build) (64-bit)
操作系统 win10

**Reply by xiaodong2008** [reply]
你好，可以提供一下截图吗

**Reply by applebluecode** [reply]
![image](https://github.com/vuejs-translations/docs-zh-cn/assets/120706957/b682428d-316e-403c-a1aa-416db1201ec1)

**Reply by xiaodong2008** [reply, marked as answer]
请参考 #919 的解决方案
